<launch>
    <!-- Arguments -->
    <!-- <arg name="obstacles_file" default="$(find mrs_visitor)/resources/obstacles.asc" /> -->
    <arg name="obstacles_file" default="$(find mrs_visitor)/resources/models/Towers/meshes/towers.dae" />
    <arg name="world_bounds_file" default="$(find mrs_visitor)/resources/world_bounds.yaml" />
    <arg name="uav_id" default="$(optenv UAV_NAME uav1)" />
    <!-- <arg name="safety_distance" default="2.0" /> -->
    <!-- <arg name="rrt_timeout" default="10.0" /> -->
    <!-- <arg name="rrt_inflation" default="0.2" /> -->

    <node pkg="mrs_visitor" name="mrs_visitor" type="node.py" output="screen">
        <!-- Configuration for the planner -->
        <!-- <rosparam file="$(find visitor)/config/config.yaml" command="load" /> -->

        <!-- Obstacles file path parameter -->
        <param name="obstacles_file" value="$(arg obstacles_file)" />
        <param name="world_bounds_file" value="$(arg world_bounds_file)" />
        <param name="uav_id" value="$(arg uav_id)" />
        <!-- <param name="safety_distance" value="$(arg safety_distance)" /> -->
        <!-- <param name="rrt_timeout" value="$(arg rrt_timeout)" /> -->
        <!-- <param name="rrt_inflation" value="$(arg rrt_inflation)" /> -->
    </node>
</launch>